cephfs/path.go

```diff
@@ -44,3 +44,68 @@ func (mount *MountInfo) RemoveDir(path string) error {
 	ret := C.ceph_rmdir(mount.mount, cPath)
 	return getError(ret)
 }
+
+// Link creates a new link to an existing file
```

Comment:
Go doc comment style encourages these to be full sentences. Thus, this needs a trailing period.